templates/workspaces/base/terraform/aad/aad.tf

```diff
@@ -124,3 +124,25 @@ resource "azuread_app_role_assignment" "workspace_owner" {
   principal_object_id = var.workspace_owner_object_id
   resource_object_id  = azuread_service_principal.workspace.object_id
 }
+
+resource "azuread_group" "workspace_owners" {
+  count        = var.create_aad_groups ? 1 : 0
+  display_name = "${var.workspace_resource_name_suffix} Workspace Owners"
```

Comment:
Since this supposed to be related to the rest of the AAD config, won't it make sense to have it named with the workspace's app id rather than the its TRE resource id?

Reply:
No, as the users need to be able to identify which workspace the group is associated with when working in Azure AD self service tooling. The workspace ID is more appropriate as this wiill be displayed in the UI.